guard.cpp

```diff
@@ -90,6 +123,8 @@ int main(int argc, char** argv)
         std::optional<std::string> deleteGuardStr;
         bool listGuardRecords = false;
         bool clearAll = false;
+        bool clearResolvedRecord = false;
```

Comment:
Unused variable
- did you ran clang?
- did you enable support in the meson.build to treat warnings as errors?

Reply:
I ran clang, looks like you reviewed old patch. In new patch set its removed.

Reply:
Please remove in the same commit - Now you can ignore. The reviewer will add comments unnecessarily by looking at the current commit/patch :)